README.md

```diff
@@ -37,7 +37,8 @@ Option                  |  Usage
 TestFramework           | One of the following test frameworks for execution: Jasmine/Mocha/Jest                | Jasmine
 DebugLogs               | Enable debug logs for JavaScript test runner                                          | false
 DebugFilePath           | Path for diagnostic logs                                                              | ""
-TestFrameworkConfigJson | Override test framework configurations (Specific to the testframework) in json format | {} 
+TestFrameworkConfigJson | Override test framework configurations (Specific to the testframework) in json format | {}
+NodeModulesPath         | Override path to node_modules                                                         | ""
 

```

Comment:
Custom path to node_modules

Reply:
yes that sound better I will change that